test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightWriterTests.cs

```diff
@@ -2031,9 +2036,15 @@ public async Task WriteEndAsync_ThrowsExceptionForBinaryStreamNotDisposed()
                 container = containerBuilder.BuildContainer();
             }
 
+            Stream stream = this.stream;

```

Comment:
To avoid confusion, how about you name this local stream `messageStream` since it's assigned to the `MessageStream` property...